gauntlet/packages/gauntlet-serum-multisig/src/commands/create.ts

```diff
@@ -1,16 +1,16 @@
 import { Result } from '@chainlink/gauntlet-core'
 import { logger, BN, prompt } from '@chainlink/gauntlet-core/dist/utils'
-import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { SolanaCommand, TransactionResponse, contracts } from '@chainlink/gauntlet-solana'
 import { PublicKey, SYSVAR_RENT_PUBKEY, Keypair } from '@solana/web3.js'
-import { CONTRACT_LIST, getContract } from '@chainlink/gauntlet-solana-contracts'
+import { CONTRACT_LIST, getContract } from '../lib/contracts'
 
 type Input = {
   owners: string[]
   threshold: number | string
 }
 
 export default class MultisigCreate extends SolanaCommand {
-  static id = 'create'
+  static id = 'create:multisig'
```

Comment:
Let's follow the same style as the other commands, `<program>:<function>` , in this case, `serum_multisig:create`
The same with the rest of exclusive msig commands (inspect, setOwners and setThreshold)